src/test/java/com/guysfromusa/carsgame/repositories/MapRepositoryTest.java

```diff
@@ -14,20 +11,22 @@
 /**
  * Created by Sebastian Mikucki, 26.02.18
  */
-@RunWith(SpringJUnit4ClassRunner.class)
-@ContextConfiguration(classes = SpringContextConfiguration.class)
-public class MapRepositoryTest {
+public class MapRepositoryTest extends BaseRepositoryTest {
 
     @Inject
     private MapRepository mapRepository;
 
     @Test
+    @DatabaseSetup("/insert-map-data.xml")
```

Comment:
@all please use one xml file per one test mehtod. then xmls are smaller and easier to understand.